style/js/shield_defs.js

```diff
@@ -384,6 +384,48 @@ export function loadShields(shieldImages) {
     },
   };
 
+  ["COL", "JEF", "MAH", "OTT", "SEN", "STA", "SUM", "TUS"].forEach(
+    // Yellow on blue pentagon
+    (county) => (shields["US:OH:" + county] = usMUTCDCountyShield)
+  );
+  ["ATH", "BEL", "GUE", "HAR", "HEN", "MOE", "PAU", "WAS", "WIL", "WYA"].forEach(
+    // White on green rectangle
+    (county) => (shields["US:OH:" + county] = roundedRectShield("#006747", "white", "white", 2, 1, null))
+  );
+  ["MED", "NOB"].forEach(
+    // White on blue rectangle
+    (county) => (shields["US:OH:" + county] = roundedRectShield("#003f87", "white", "white", 2, 1, null))
+  );
+  ["TRU", "VIN"].forEach(
+    // Black on yellow rectangle
+    (county) => (shields["US:OH:" + county] = roundedRectShield("#ffcd00", "black", "black", 2, 1, null))
+  );
+  shields["US:OH:ASD"] = {
+    backgroundImage: [
+      shieldImages.shield40_us_oh_asd,
+    ],
+    textColor: "#006747",
+    padding: {
+      left: 6,
+      right: 3,
+      top: 4,
+      bottom: 7,
+    },
+  };
```

Comment:
Four-digit Ashland County road numbers are practically illegible, but I’ve already distorted the county shape to be barely recognizable. The number is practically illegible in real life anyways.

[<img src="https://upload.wikimedia.org/wikipedia/commons/8/87/Ashland_County_Route_1095_OH.svg" height="64" alt="Ashland Co. 1095">](https://commons.wikimedia.org/wiki/File:Ashland_County_Route_1095_OH.svg)